### ISSUE TITLE
About contributors in the doc.

### ISSUE THREAD

**Issue by JiuLing-zhang** [dotnet/docs#37480]
**Issue description**

I submitted [37431](https://github.com/dotnet/docs/pull/37431/files#diff-b074109b6599f02787017f9c5f913ca6db06ab3d6a8c53c4df197c5b76bbea7e) [37432](https://github.com/dotnet/docs/pull/37432/files#diff-2f6c5de8cd01ac7172643b52229b598da13bf370a3e9cdc205636c813acf8b90) in this [page](https://learn.microsoft.com/en-us/dotnet/standard/parallel-programming/task-based-asynchronous-programming), this is my first commit, why can’t I see myself in the contributor list on the page?

![image](https://github.com/dotnet/docs/assets/78294929/884aee2b-c6a1-4440-ac64-be0dd2e825e9)

**Comment by BillWagner** [comment]
Hi @JiuLing-zhang 

I don't know of a limit on the list of contributors. I'm working to find what the issue is and try to get it fixed.

**Comment by JiuLing-zhang** [comment]
> Hi @JiuLing-zhang
> 
> I don't know of a limit on the list of contributors. I'm working to find what the issue is and try to get it fixed.

Hi @BillWagner , Any updates?

**Comment by BillWagner** [comment]
HI @JiuLing-zhang 

I did just get an answer:  It's a current limitation in the platform. You PRs updated the sample that's included on the page, but didn't change the source file for the article.

I've filed a request on the team's internal board to get that fixed.

In the meantime, the tool we use to generate the [What's new in docs](https://learn.microsoft.com/en-us/dotnet/whats-new/dotnet-docs-mod0) pages does find all contributions. You'll be recognized there when the page for October is generated.

I'll close this as we're tracking the enhancement internally.
